src/openms/source/ANALYSIS/ID/SimpleSearchEngineAlgorithm.cpp

```diff
@@ -47,12 +47,19 @@ using namespace std;
 
 namespace OpenMS
 {
+ /**
```

Comment:
we mainly put the comments in the header file (I am not sure if doxygen picks them up from the source file)